sbws/lib/v3bwfile.py

```diff
@@ -1201,27 +1242,27 @@ def is_min_perc(self):
 
     @property
     def sum_bw(self):
-        return sum([l.bw for l in self.bw_lines])
+        return sum([l.bw for l in self.bw_lines if getattr(l, 'bw', None)])
```

Comment:
Maybe use hasattr in these checks here instead of getattr?  I think it is closer to what you mean.

Reply:
i forgot hasattr exists, yeah, changed, also all the other lines, because i don't want to take them into account if they don't have the attribute.